docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -36,73 +44,192 @@ $ kubectl config get-contexts
+   --auth-server=192.0.2.255:3025
+Please note:
+  - This invitation token will expire in 30 minutes
+  - 192.0.2.255:3025 must be reachable from the new node
 ```
 
+On the host where you are running the Teleport Kubernetes Service, create a
+file called `/tmp/token` that consists only of your token:
+
+```code
+$ echo <Var name="join-token" /> | sudo tee /tmp/token
+```
+
+### Install the Teleport Kubernetes Service
+
+Run the following commands on the host where you will install the Teleport
+Kubernetes Service:
+
+<Tabs dropdownView dropdownCaption="Teleport Edition">
+<TabItem label="Self-Hosted" scope={["oss", "enterprise"]}>
+
+(!docs/pages/includes/install-linux.mdx!)
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope="cloud">
+
+(!docs/pages/includes/install-agent-cloud.mdx service="the Teleport Kubernetes Service"!)
+
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Cloud">
+</Tabs>
+
+### Configure the Teleport Kubernetes Service
 
-On the host you will use to run the Teleport Kubernetes Service, add the
-following to `teleport.yaml`.
+On the host where you will run the Teleport Kubernetes Service, create a file at
+`/etc/teleport.yaml` with the following content:
 
 ```yaml
-# ...
+version: v3
+teleport:
+  join_params:
+    token_name: "/tmp/token"
+    method: token
+  proxy_server: "teleport.example.com:3080"
+auth_service:
+  enabled: off
+proxy_service:
+  enabled: off
+ssh_service:
+  enabled: off
 kubernetes_service:
-  enabled: yes
-  listen_addr: 0.0.0.0:3027
-  # Replace this path with the actual path for your generated kubeconfig
-  kubeconfig_file: "path/to/kubeconfig"
+  enabled: "yes"
+  kubeconfig_file: "/var/lib/teleport/kubeconfig"
+  labels:
+    "region": "us-east1"
 ```
 
-</TabItem>
-</Tabs>
+Edit `/etc/teleport.yaml` to replace `teleport.example.com:3080` with the host
+and port of your Teleport Proxy Service or Teleport Cloud tenant, e.g.,
+`mytenant.teleport.sh:443`.
 
 <Admonition type="warning" title="Warning" >
-  When using `kubeconfig_file`, EKS users may need to replace illegal characters in the `context` names.
-  Supported characters are alphanumeric characters, `.`, `_`, and `-`. EKS typically includes `:` and `@`
-  in their `kubeconfig` which are not allowed in Teleport.
+
+When using `kubeconfig_file`, Amazon EKS users may need to replace illegal
+characters in the `context` names. Supported characters are alphanumeric
+characters, `.`, `_`, and `-`. EKS typically includes `:` and `@` in their
+`kubeconfig` files, which are not allowed in Teleport.
+
 </Admonition>
 
+### Start the Teleport Kubernetes Service
+
+On the host where you will run the Teleport Kubernetes Service, execute the
+following command, depending on whether you installed Teleport using a package
+manager or via a TAR archive:
+
+<Tabs>
+<TabItem label="Package Manager">
+
+```code
+$ sudo systemctl start teleport
+```
+</TabItem>
+<TabItem label="TAR archive">
+```code
+$ sudo teleport start
```

Comment:
Hrm, I'm not sure we want to recommend people running Teleport in the foreground like this in any guides beyond "trying Teleport out". Can we direct people towards using `sudo teleport install systemd` and then using `sudo systemctl start teleport` ?